.travis.yml

```diff
@@ -2,29 +2,26 @@ sudo: required
 dist: trusty
 language: python
 
-python:
-    - "2.7"
-    - "3.4"
-    - "3.5"
-    - "3.6"
+matrix:
+    include:
+        - python: 3.5
+        - python: 3.6
+        - python: 3.7
+          dist: xenial
+          sudo: true
 
 before_install:
     - sudo apt-get update
     - sudo updatedb
-    - if [ $TRAVIS_PYTHON_VERSION == "2.7" ]; then
-        wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
-      else
-        wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
You need to install miniconda, you can't remove this line.
The Travis checks are failing for this reason.

Reply:
Fixed